### ISSUE TITLE
[gometa] Documentation

### ISSUE THREAD

**Issue by stp-ip** [txtdirect/txtdirect#182]
**Is this a BUG REPORT or FEATURE REQUEST?**:
docs

# gometa:
```
schema.resumic.test.              IN A 127.0.0.1
_redirect.schema.resumic.test. IN TXT "v=txtv0;to=https://github.com/resumic/schema;type=gometa"
```
go get -u schema.resumic.test -> downloads https://github.com/resumic/schema

# gometa + path:
```
go.resumic.test.              IN A 127.0.0.1
_redirect.go.resumic.test. IN TXT "v=txtv0;to=https://about.resumic.org;type=path"
_redirect.schema.go.resumic.test. IN TXT "v=txtv0;to=https://github.com/resumic/schema;type=gometa"
```
go get -u go.resumic.test/schema -> downloads https://github.com/resumic/schema
Visiting go.resumic.test -> redirect to https://about.resumic.org
go get -u go.resumic.test -> failure aka no meta tag as expected

Both methods also work for go imports:
import "go.resumic.test/schema/cmd"

**Comment by erbesharat** [comment]
Fixed in txtdirect/website#16

**Comment by stp-ip** [comment]
Closing in anticipation to the PR merging.
